machine.go

```diff
@@ -424,15 +391,53 @@ func (m *Machine) setupLogging(ctx context.Context) error {
 	}
 
 	resp, err := m.client.PutLogger(ctx, &l)
-	if err == nil {
-		m.logger.Printf("Configured VMM logging to %s, metrics to %s: %s",
-			m.cfg.LogFifo, m.cfg.MetricsFifo, resp.Error())
+	if err != nil {
+		return err
 	}
-	return err
+
+	m.logger.Printf("Configured VMM logging to %s, metrics to %s: %s",
+		m.cfg.LogFifo, m.cfg.MetricsFifo, resp.Error())
```

Comment:
Let's drop this to Debug severity, and also drop `resp.Error()`, since it won't contain anything meaningful.